### DISCUSSION TITLE
New service check-if-email-exists

### DISCUSSION THREAD

**Discussion by pierre-H** [coollabsio/coolify#4590] [Service Template Requests]
Proposition of  a great service :

[https://github.com/reacherhq/check-if-email-exists](check-if-email-exists)

**Comment by MichaelTendoSsemwanga** [comment]
+1

**Comment by ztane** [comment]
The link uri I believe should have been [https://github.com/reacherhq/check-if-email-exists](https://github.com/reacherhq/check-if-email-exists)

**Comment by albertorizzi** [comment]
I tried this compose, but it doens't work

```
version: '3.8'
services:
  reacher:
    image: 'reacherhq/backend:latest'
    ports:
      - '8081:8081'
    environment:
      - 'RCH__HTTP__HOST=${RCH__HTTP__HOST:-0.0.0.0}'
      - 'RCH__HTTP__PORT=${RCH__HTTP__PORT:-8081}'
      - 'RCH__HEADER_SECRET=${RCH__HEADER_SECRET:-}'
      - 'SERVICE_FQDN_REACHER_8081=${SERVICE_FQDN_REACHER_8081}'
    restart: unless-stopped
    healthcheck:
      test:
        - CMD-SHELL
        - 'curl -f http://0.0.0.0:8081 || exit 1'
      interval: 10s
      timeout: 10s
      retries: 5

**Comment by TheMikeyRoss** [comment]
Going through their docs [here](https://docs.reacher.email/self-hosting/install#prerequisites) it seems it's not self-hostable in the traditional sense. It requires an account in reacher.email and an api key just so you can host it locally in your computer.

but I could be wrong.

**Reply by albertorizzi** [reply]
Ok, thank you, so it's not useful
